.github/workflows/release-integration.yml

```
@@ -91,4 +98,9 @@ jobs:
           RELEASE_TAG: ${{ steps.create-release-package.outputs.tag-name }}
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
-          gh release create "$RELEASE_TAG" --draft --notes-file "${{ env.RELEASE_NOTES_FILE }}" --title "$RELEASE_TAG"
+          PARAMS=""
+          if [[ "${CREATE_DRAFT_RELEASE}" == "true" ]]; then
+            PARAMS="--draft "
+          fi
+
+          gh release create "$RELEASE_TAG" $PARAMS --notes-file "${{ env.RELEASE_NOTES_FILE }}" --title "$RELEASE_TAG"
```

You could directly make use of a boolean input (with a default value) by doing the following:

```suggestion
          gh release create "$RELEASE_TAG" --draft=${{ inputs.draft }} --notes-file "${{ env.RELEASE_NOTES_FILE }}" --title "$RELEASE_TAG"
```